From a94c7d4917f54bb9929d9c247149eb0c400aac36 Mon Sep 17 00:00:00 2001
From: Aditya Kumar Singh <quic_adisi@quicinc.com>
Date: Wed, 10 Nov 2021 16:03:56 +0530
Subject: [PATCH] fix sae and sae-mixed support

sae and sae-mixed support due to commit id
22476ff99ea2c233e342dfe2bdff506e54278f03 in upstream
is conflicting the already existing sae and sae-mixed
support.

This fixes this issue.

Signed-off-by: Aditya Kumar Singh <quic_adisi@quicinc.com>
---
 scripts/netifd-wireless.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/netifd-wireless.sh b/scripts/netifd-wireless.sh
index 2dca493..b604e52 100644
--- a/scripts/netifd-wireless.sh
+++ b/scripts/netifd-wireless.sh
@@ -249,10 +249,10 @@ wireless_vif_parse_encryption() {
 		wpa3*)
 			auth_type=eap192
 		;;
-		psk3-mixed*|sae-mixed*)
+		psk3-mixed*)
 			auth_type=psk-sae
 		;;
-		psk3*|sae*)
+		psk3*)
 			auth_type=sae
 		;;
 		*psk*)
-- 
2.7.4

